### PR TITLE
Add "Open Containing Folder as Workspace" command

### LINKED ISSUE #3028: [Feature] [Context menu] Open directory as new "Folder as workspace" (as Project)
Context menu on folder or in folder's background > Open as new "Folder as workspace" (name it whatever you like; I will name it "Open as new project". Yes - it would be nice to have Projects, like in every modern editors).
It will open that folder in notepad++ as new instance of app, without files that are opened in other instances.
And project file will be automatically generated in that directory.

My feature is related to this:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/1646

### PR DESCRIPTION
**Pull request by donho**
and add this command in context menu of tabs.

Close #3028

### CODE REVIEW

**Comment by Yaron10** [comment]
@donho,

:+1: 
That's a useful addition. Thank you.

File: PowerEditor/installer/nativeLang/english.xml [hunk 1]
```diff
@@ -75,6 +75,7 @@
                     <Item id="41002" name="&amp;Open"/>
                     <Item id="41019" name="Explorer"/>
                     <Item id="41020" name="cmd"/>
+                    <Item id="41025" name="Folder as Workspace"/>
```
Comment (rddim) [inline PowerEditor/installer/nativeLang/english.xml:78]: We have already the same under `View` > `Folder as Workspace`

**Comment by Yaron10** [comment]
@donho,

Can you please try the following case?

Open folder `A` as Workspace.
Close the FAW panel and close NPP.
Restart NPP (no session, "New 1).
Open Containing Folder as Workspace.

Result:
Folder `A` is removed from FAW.

Thank you.

File: PowerEditor/installer/nativeLang/english.xml (hunk 1 above)
Comment (donho) [reply PowerEditor/installer/nativeLang/english.xml:78]: So? It's not the same command.

**Comment by donho** [comment]
@Yaron10 
My laziness of developer told me it's OK. Then you came out and shouted "NO, IT'S NOT OK!" on my face...
OK, I will see what I can do about it.

**Comment by Yaron10** [comment]
@donho,

Juggling with all the issues/PRs is a full-time job. Laziness forgiven. :)
Thank you for your work. :+1:

**Comment by donho** [comment]
@Yaron10 
Fixed, please try it - this one is even more accurate.
Thank you for testing and point out the problem.

**Comment by Yaron10** [comment]
@donho,

:+1: 
It works like a charm. Thank you.
The current file being selected is an important bonus. :)

**Pull request closed without merging**